package.json

```diff
@@ -92,6 +92,7 @@
 		"emittery": "^0.3.0",
 		"empower-core": "^0.6.1",
 		"equal-length": "^1.0.0",
+		"esm": "^3.0.22",
```

Comment:
The `package-lock.json` diff seems surprisingly large given this one change. Might be good to `git checkout master package-lock.json` and `npm i` again. (At least, I think that's the right `git` incantation.)

Reply:
Ok, better now: from 6,951 to 266